codestodians/food-basket-api
# Fix production db config

#### What does this PR do?
- This PR adds a fix for the db prod config

#### Description of Task to be completed?
- The database setting(name, user, password, e.t.c) is set to use the postgres db hosted on heroku

#### How should this be manually tested?
- Using the heroku hosted graphql endpoint, create a user.
- Notice that the user is setup in the database.

#### Any background context you want to provide?
- The db was previoulsy used locally. This PR allows the use of the heroku db.

#### What are the relevant pivotal tracker stories?
[#154166093](https://www.pivotaltracker.com/story/show/154166093)

#### Screenshots (if appropriate)

#### Additional info:
To setup the db locally(pg-admin)
- Create new server
- Fill info with pinned db credentials
- Select the Server, right click -> properties, Advanced tab.
- In 'DB restriction' type in the name of your database(s) enclosed in single quotes and seperated by spaces.
- That should work.
